### PR TITLE
Refs [TASK][36590] Data HomeScreen

### PR DESCRIPTION
**Pull request by manhhung23399**
## Related Tickets
- [#Task 36590: Data HomeScreen](https://edu-redmine.sun-asterisk.vn/issues/36590)
## WHAT
- bind data spinner
- create xbox fragment
## Evidence (Screenshot or Video)
image

## Review Checklist

Category | View Point | Description | Expected Reviewer Answer | Self review | Reviewer2 (name)
--- | --- | --- | --- | --- | ---
Conventions | Does the code follow Sun* coding style and coding conventions? | https://github.com/framgia/coding-standards/tree/master/eng/android | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Redmine | Does the ticket follow Sun* Redmine working process?  | https://github.com/framgia/Training-Guideline/blob/master/WorkingProcess/redmine/redmine.md| YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
Documentation | Is there any incomplete code? If so, should it be removed or flagged with a suitable marker like ‘TODO’? |  | YES |<li>- [ ] yes</li>|<li>- [ ] yes</li>  
## Notes (Optional)
*(Impacted Areas in Application(List features, api, models or services that this PR will affect))*

*(List gem, library third party add new)*

*(Other notes)*

### CODE REVIEW

File: app/src/main/java/com/sun/gamevui/data/GenreRemoteDataSource.kt [hunk 1]
```diff
@@ -0,0 +1,9 @@
+package com.sun.gamevui.data
+
+import com.sun.gamevui.data.remote.ApiService
+
+class GenreRemoteDataSource(
+    private val apiService: ApiService
+) : GenreDataSource {
```
Comment (tuannm-1276) [inline app/src/main/java/com/sun/gamevui/data/GenreRemoteDataSource.kt:7]: Genre chỉ remote thôi hả em

File: app/src/main/java/com/sun/gamevui/data/remote/ApiConfig.kt [hunk 2]
```diff
@@ -8,4 +8,5 @@ object ApiConfig {
     const val BASE_PS = "2"
     const val BASE_XBOX = "3"
     const val BASE_PARENT_PLATFORM = "parent_platforms"
+    const val BASE_GENRE = "genres"
```
Comment (tuannm-1276) [inline app/src/main/java/com/sun/gamevui/data/remote/ApiConfig.kt:14]: có thể comment cách cấu tạo của các url vào class này để ng khác dễ hình dùng hơn

**Review by thonc-2135 — APPROVED** [review]

File: app/src/main/java/com/sun/gamevui/data/GenreRemoteDataSource.kt (hunk 1 above)
Comment (manhhung23399) [reply app/src/main/java/com/sun/gamevui/data/GenreRemoteDataSource.kt:7]: Vâng a, genre e lấy từ remote về thôi ạ

File: app/src/main/java/com/sun/gamevui/data/remote/ApiConfig.kt (hunk 2 above)
Comment (manhhung23399) [reply app/src/main/java/com/sun/gamevui/data/remote/ApiConfig.kt:14]: như này hả a
vd: 
// https://api.rawg.io/api/games?genres=&key=

**Pull request merged**